pbjam/l1models.py

```diff
@@ -1085,19 +1085,18 @@ def setPriors(self,):
                                                  self.DR.logpdf[i], 
                                                  self.DR.cdf[i])
 
-        AddKeys = [k for k in self.variables if k in self.addPriors.keys()]
-
-        self.priors.update({key : self.addPriors[key] for key in AddKeys})
-
         self.priors['q'] = dist.uniform(loc=0.01, scale=0.6)
-
         # Core rotation prior
         self.priors['nurot_c'] = dist.uniform(loc=-2., scale=3.)
-
         self.priors['nurot_e'] = dist.uniform(loc=-2., scale=2.)
 
         # The inclination prior is a sine truncated between 0, and pi/2.
-        self.priors['inc'] = dist.truncsine() 
+        self.priors['inc'] = dist.truncsine()
```

Comment:
I think for changing the order of things here you'll need to check that it doesn't influence the sampling. 

I remember having problems with that at some point, where I thought using dictionaries should have solved this, but apparently it didn't. 

Just so we aren't sampling nurot_e when we think it's something else like eps_g.